### PR TITLE
Adding unit tests for adapter

### PR DESCRIPTION
**Pull request by VajiraPrabuddhaka**
### Purpose
Added unit tests for adapter (envoyconf & model).
Removed some unwanted methods.
Cleansed up unused dependencies with `go mod tidy`.

### Issues

Fixes #

### Automation tests
 - Unit tests added: Yes
 - Integration tests added: Yes/No

### Tested environments

Not Tested

---
#### Maintainers: Check before merge
- [x] Assigned 'Type' label
- [x] Assigned the project
- [ ] Validated respective github issues
- [ ] Assigned milestone to the github issue(s)

### CODE REVIEW

File: adapter/internal/oasparser/envoyconf/routes_with_clusters_test.go [hunk 1]
```diff
@@ -259,3 +263,19 @@ func testCreateRoutesWithClustersWebsocket(t *testing.T, apiYamlFilePath string)
...
+	apiJsn, conversionErr := utills.ToJSON(apiYamlByteArr)
+	assert.Nil(t, conversionErr, "YAML to JSON conversion error : %v"+apiYamlFilePath)
+	mgwSwagger := operator.GetMgwSwaggerWebSocket(apiJsn)
+	//routes, clusters, _ := envoy.CreateRoutesWithClusters(mgwSwagger, nil)
+	routes, clusters, _ := CreateRoutesWithClusters(mgwSwagger, nil)
+	t.Log(routes)
+	t.Log(clusters)
+}
+
+// func TestCreateCluster()
```
Comment (chashikajw) [inline adapter/internal/oasparser/envoyconf/routes_with_clusters_test.go:281]: shall we remove this

Comment (chashikajw) [inline adapter/internal/oasparser/envoyconf/routes_with_clusters_test.go:281]: shall we remove this?

File: adapter/internal/oasparser/envoyconf/routes_with_clusters_test.go [hunk 2]
```diff
@@ -259,3 +260,17 @@ func testCreateRoutesWithClustersWebsocket(t *testing.T, apiYamlFilePath string)
...
+	apiYamlFilePath := config.GetMgwHome() + "/../adapter/test-resources/envoycodegen/api.yaml"
+	apiYamlByteArr, err := ioutil.ReadFile(apiYamlFilePath)
+	assert.Nil(t, err, "Error while reading the api.yaml file : %v"+apiYamlFilePath)
+	apiJsn, conversionErr := utills.ToJSON(apiYamlByteArr)
+	assert.Nil(t, conversionErr, "YAML to JSON conversion error : %v"+apiYamlFilePath)
+	mgwSwagger := operator.GetMgwSwaggerWebSocket(apiJsn)
+	//routes, clusters, _ := envoy.CreateRoutesWithClusters(mgwSwagger, nil)
+	routes, clusters, _ := envoy.CreateRoutesWithClusters(mgwSwagger, nil)
+	t.Log(routes)
+	t.Log(clusters)
```
Comment (chashikajw) [inline adapter/internal/oasparser/envoyconf/routes_with_clusters_test.go:275]: What are these logs? Do we need these for tests ?

File: adapter/internal/oasparser/model/swagger_test.go [hunk 3]
```diff
@@ -0,0 +1,78 @@
+/*
+ *  Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (chashikajw) [inline adapter/internal/oasparser/model/swagger_test.go:2]: If this is a new file, change this to 2021

File: adapter/internal/oasparser/model/swagger_test.go [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+)
+
+func TestSetInfoSwaggerWebSocket(t *testing.T) {
+
+	type setInfoSwaggerWebSocketTestItem struct {
+		input   model.MgwSwagger
+		apiData map[string]interface{}
+	}
+
+	//var dataItems []setInfoSwaggerWebSocketTestItem
```
Comment (chashikajw) [inline adapter/internal/oasparser/model/swagger_test.go:37]: Remove commnetted lines

File: adapter/internal/oasparser/operator/operator_test.go [hunk 5]
```diff
@@ -140,6 +142,84 @@ func TestMgwSwaggerWebSocketSand(t *testing.T) {
...
+	if err != nil {
+		t.Errorf("Unable to read test-resources")
+	}
+
+	for _, f := range files {
+		if strings.HasSuffix(f.Name(), ".yaml") {
+			testGetOpenAPIVersionAndJSONContent(t, apiYamlFilePath+"/"+f.Name())
+		}
+	}
+
```
Comment (chashikajw) [inline adapter/internal/oasparser/operator/operator_test.go:160]: Remove extra lines

File: adapter/internal/oasparser/envoyconf/routes_with_clusters_test.go (hunk 2 above)
Comment (VajiraPrabuddhaka) [reply adapter/internal/oasparser/envoyconf/routes_with_clusters_test.go]: removed logs and added assertions for those

**Comment by VajiraPrabuddhaka** [comment]
Review changes has been added

**Review by Rajith90 — APPROVED** [review]

**Pull request merged**